lib/sources/node.js

```diff
@@ -33,12 +36,30 @@ NodeSource.prototype.update = function(done) {
     if (!res.text) return done(new Error('No response'), false);
     if (res.status !== 200) return done(new Error('Bad response'), false);
 
-    this._parse(res.text)
-    done(undefined, true);
+      responseText += res.text;
+
+      agent
+        .get(this.candidateUrl)
+        .timeout(TIMEOUT)
+        .end(parseResponse.bind(this));
+
+      function parseResponse(err, res) {
```

Comment:
There's a lot of duplicated code in this PR. Why not download from both URLs, concatenate bodies (maybe separating them with a new line) and have `parseResponse` check both responses?

Also, note that `/download/rc/` is a location of RCs but in the future there will also be betas/alphas, it should be decided what to do with them. For example, supporting nightlies would most likely be too much as they will arrive extremely frequently.

(NOTE: I'm not a maintainer of this module so my comments may be overruled :))